docs/use/dapp-staking/for-devs/index.md

```diff
@@ -1,15 +1,98 @@
+import Figure from "/src/components/figure"
+
 # For Developers
 
 ## Overview
 
-Are you a product owner and want to list your dApp in our staking mechanism to earn rewards? You have to make sure you meet the requirements. Read more about those by clicking on the links below.
+Discover [dApp Staking](/docs/learn/dapp-staking/), a unique mechanism that allows communities to support their favorite teams by staking ASTR or SDN with them. When staking ASTR or SDN on a dApp, users not only support the development of innovative apps but also receive staking rewards from inflation.
+
+You can educate yourself more by reading the technical documentation about [dApp Staking](/docs/learn/dapp-staking/).
+
+**Are you a product owner and want to list your dApp in our staking mechanism to earn rewards?**  
+
+Please take the following parameters into consideration and make sure you meet the [requirements](/docs/use/dapp-staking/for-devs/requirements.md):
+
+- To register a dApp for dApp staking you need to have 100 SDN tokens or 1000 ASTR tokens depending on the network you would like to join as a project. dApp staking on our testnet Shibuya requires 100 SBY.
+- **You must claim your rewards to receive them. We recommend that you claim your rewards at the beginning of each new period.**
```

Comment:
Let's recommend weekly or 2-3 times a week.
We want to avoid rewards accumulating for the entire period because multiple claim calls might be required to cover the whole time span.